### ISSUE TITLE
Create and Train ML Model

### ISSUE THREAD

**Issue by AdityaPunetha** [upes-open/OSC-Medical-Analysis-Application-Using-ML#4]
Within the ML model folder, create a jupyter notebook ( .ipynb extention ) named Training.ipynb and create an CNN model in it. Train it on the [Kaggle Chest X-Ray Images Pneumonia dataset](https://www.kaggle.com/datasets/paultimothymooney/chest-xray-pneumonia) and save it with .h5 extension. the model should be named PneumoniaClassifierV1.h5, please mention its accuracy in the pull request as well. Upload the model in a new folder called models within ML Model folder

The model would be a CNN built through Keras and TensorFLow, the architecture of the model is left to the contributor, it is recommended to use a combination of convolution, maxpooling, dense and dropout layer to ensure maximum accuracy and efficiency. Since it is a binary classifier, Binary cross entropy loss function is recommended.
